src/index.js

```diff
@@ -90,7 +90,9 @@ let mockFunction: MockFunctionType<*, *>, test: any;
 if (typeof it !== 'undefined') {
   // Surface snapshot testing
   // $FlowFixMe
-  assert.matchSnapshot = tree => expect(tree).toMatchSnapshot();
+  assert.matchSnapshot = (tree, snapshotName) =>
+    // $FlowFixMe
```

Comment:
Why this $FlowFixMe?

Reply:
Without this, `flow` was complaining that `expect` could not be found.

Reply:
Any idea on how to better address that?

Reply:
Ah, ok, I think this was maybe pre-existing then. Nice!